gneiss/plot/_plot.py

```diff
@@ -387,10 +413,15 @@ def dendrogram_heatmap(output_dir: str, table: pd.DataFrame,
                  'present in this tree.')},
     parameter_descriptions={
         'metadata': ('Metadata to group the samples. '),
-        'ndim': 'Number of dimensions to highlight.'},
+        'ndim': 'Number of dimensions to highlight.',
```

Comment:
So this is always just highlighting the `n` most basal splits in the tree, right? 

In the future do you want to expose selective highlighting of e.g. most significant splits, or prefer to keep those int he boxplot visualizations?